src/compile/legend/properties.ts

```diff
@@ -15,15 +15,3 @@ export function values(legend: Legend) {
   }
   return vals;
 }
-
-export function type(t: Type, channel: Channel, scaleType: ScaleType): 'gradient' {
```

Comment:
I'm curious why this should be removed.  You probably have a good reason, but better explain too. :)

Reply:
Now, the type is inferred from the scale type and we do not override it anymore.